pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "prediction-market-agent"
+version = "0.1.0"
+description = ""
+authors = ["Gnosis"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.10"
+python-dotenv = "*"
+openai = "*"
+langchain = "*"
+langchain-experimental = "*"
+numexpr = "*"
+pyautogen = "*"
+beautifulsoup4 = "*"
+serpapi = "*"
+google-search-results = "*"
+pytest = "*"
+llama-index = "~0.9.0"
+metagpt = "*"
+duckduckgo-search = "*"
+# crewai = "*"  # Conflicts with metagpt due to langchain, commented out as metagpt seems to have bigger traction, install manually in the environment for usage.
+
```

Comment:
I'm less familiar with poetry vs pip. Do you still get the local package installed with `poetry install`? Or do you need something like this here:
```
[tool.poetry.dev-dependencies]
prediction-market-agent = { path = ".", develop = true }
```
(added when you do `poetry add --dev .`) for that to work?

Reply:
Poetry won't allow to lock it with Crewai because of conflicts. However, once you have the poetry environment setup, you can do `pip install crewai` to force installation.

I'm not sure what's the best way to proceed here, as we are experimenting with a lot of experimental packages. If we find out we want CrewAi instead of Metagpt we can comment out Metagpt instead.

Reply:
Ah sorry, I'm not referring to the commented out crewai package here. I just mean does doing `poetry install` also install the local `prediction_market_agent` package? If not then running `python main.py` wouldn't work, so I guess you've tested that!

Reply:
Aaa sorry! I somehow auto-assumed the comment was for the `crewai` and didn't notice you actually wrote `prediction-market-agent` in the code block.

Yes, it does, `python main.py` works as expected

Reply:
Ok sweet